src/main/java/de/tum/in/ase/parser/strategy/SwiftLintParser.java

```diff
@@ -0,0 +1,43 @@
+package de.tum.in.ase.parser.strategy;
+
+import nu.xom.Document;
+
+import de.tum.in.ase.parser.domain.Issue;
+import de.tum.in.ase.parser.domain.Report;
+
+class SwiftLintParser extends CheckstyleFormatParser {
+
+    @Override
+    public Report parse(Document doc) {
+        Report report = new Report(StaticCodeAnalysisTool.SWIFTLINT);
+        extractIssues(doc, report);
+        return report;
+    }
+
+    /**
+     * Extracts and sets the rule and the category given the check's package name.
+     *
+     * @param issue issue under construction
+     * @param errorSource package like com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocPackageCheck. The first
+     *                    segment after '.checks.' denotes the category and the segment after the rule. Some rules do
+     *                    not belong to a category e.g. com.puppycrawl.tools.checkstyle.checks.NewlineAtEndOfFileCheck.
+     *                    Such rule will be grouped under {@link #CATEGORY_MISCELLANEOUS}.
+     */
```

Comment:
Update the doc with a real example of SwiftLint

Reply:
oh sorry. I did that actually but forgot it while refactoring the code -.-